configure.ac

```diff
@@ -6,7 +6,9 @@ define(_CLIENT_VERSION_REVISION, 99)
 define(_CLIENT_VERSION_BUILD, 0)
 define(_CLIENT_VERSION_IS_RELEASE, false)
 define(_COPYRIGHT_YEAR, 2016)
-AC_INIT([Bitcoin Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin])
+define(_COPYRIGHT_HOLDERS,[The %s developers])
```

Comment:
Yes, it's easy to do anyway, if you really want to, but it should be as hard as possible to do so *accidentally*.
E.g. I strongly feel it should at least involve changing the source code, not just the build metadata.
*Adding* potential copyright holders in the build metadata is fine with me, although those in turn may have the same concern.